### ISSUE TITLE
converting pb models for tfjs1.0.0

### ISSUE THREAD

**Issue by alex77t** [tensorflow/tfjs#1396]
I've tried to convert pretrained model from here - http://download.tensorflow.org/models/deeplabv3_mnv2_pascal_trainval_2018_01_29.tar.gz

using both approaches as described in #1379 , however in both cases when i try to load the model in the browser i;m getting the error below. 

Uncaught (in promise) Error: Based on the provided shape, [1,1,320,256], the tensor should have 81920 values but has 2295
    at d (tfjs@1.0.0:2)
    at Te (tfjs@1.0.0:2)
    at i (tfjs@1.0.0:2)
    at Object.Pl [as decodeWeights] (tfjs@1.0.0:2)
    at t.<anonymous> (tfjs@1.0.0:2)
    at tfjs@1.0.0:2
    at Object.next (tfjs@1.0.0:2)
    at a (tfjs@1.0.0:2)

here's the parameters used for tensorflowjs_converter
tensorflowjs_converter \
     --input_format=tf_frozen_model \
     --output_node_names='SemanticPredictions' \
     --saved_model_tags=serve \
     --output_json=true \
     deeplabv3_mnv2_pascal_trainval/frozen_inference_graph.pb \
     deeplabv3_mnv2_pascal_trainval/webtf

**Comment by pyu10055** [comment]
@alex77t I have tried the converted model, it runs fine on my desktop.
Can you share your code that makes the prediction?
My input tensor is an image tensor of shape [1, 224, 224, 3].

**Comment by alex77t** [comment]
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.0.0"></script>
  <script type="module">
     const MOBILENET_MODEL_PATH ='models/webtf/model.json';
     const runIt = async () =>{
        await tf.loadGraphModel(MOBILENET_MODEL_PATH);
       }
    runIt();
</script>


in my model.json i'm indeed seeing references to 1,1,320,256  - i tried with version 0.8.0 on both ubuntu and windows 10 machines. 

does your converter parameters exactly like mine? 

{"modelTopology": {"node": [{"name": "image_pooling/Conv2D/merged_input", "op": "Const", "attr": {"value": {"tensor": {"dtype": "DT_FLOAT", "tensorShape": {"dim": [{"size": "1"}, {"size": "1"}, {"size": "320"}, {"size": "256"}]}}}, "dtype": {"type": "DT_FLOAT"}}}, {"name": "image_pooling/BatchNorm/FusedBatchNorm/Offset", "op": "Const", "attr": {"value": {"tensor": {"dtype": "DT_FLOAT", "tensorShape": {"dim": [{"size": "256"}]}}}, "dtype": {"type": "DT_FLOAT"}}},

**Comment by pyu10055** [comment]
What is inside you runIt() method?

The first node of the graph is a constant node, which is not the input node.
The input node should be the node with Placeholder op:

{"attr": {"dtype": {"type": "DT_UINT8"}, "shape": {"shape": {"dim": [{"size": "1"}, {"size": "-1"}, {"size": "-1"}, {"size": "3"}    ]}}}, "name": "ImageTensor", "op": "Placeholder"}

**Comment by alex77t** [comment]
const runIt = async () =>{ await tf.loadGraphModel(MOBILENET_MODEL_PATH); } 

only the call to the tf.loadGraphModel method. 

>The input node should be the node with Placeholder op:
Is it possible to post model.json you have got? I:m attaching mine... 

[model.zip](https://github.com/tensorflow/tfjs/files/2981015/model.zip)

**Comment by pyu10055** [comment]
@alex77t If the failure happens during model loading, not at the inference time, I suspect that your server is not serving the models properly.
Can you check the network tab of your browser dev tool, see if there are any errors there. If you can post a screen capture would be great. Thanks

**Comment by alex77t** [comment]
Hi @pyu10055 , you're right - indeed the shard files were not loading correctly because of my local webserver configuration...  apologies for causing confusion and wasting your time on this. 
This ticket can be closed.
